src/software/pipeline/main_meshing.cpp

```diff
@@ -481,6 +520,19 @@ int aliceVision_main(int argc, char* argv[])
                         const double height = hexah[0].z - hexah[4].z;
 
                         ALICEVISION_LOG_INFO("bounding Box : length: " << length << ", width: " << width << ", height: " << height);
+
+                        // Save bounding box
```

Comment:
I'm not sure we should be saving this bounding box with a custom txt format.
A more standard mesh format (OBJ for instance) would probably be more suited as it would allow opening the bounding box in any 3D software.

Reply:
Good remark, but easier to use the same parameters to represent the bbox everywhere.